package.json

```diff
@@ -20,13 +19,39 @@
   },
   "homepage": "https://github.com/prateekbh/sw-generator#readme",
   "devDependencies": {
+    "@types/workbox-sw": "3.2.0",
+    "chai": "4.2.0",
+    "chromedriver": "^2.42.0",
+    "husky": "^1.1.2",
     "lint-staged": "7.3.0",
+    "mocha": "5.2.0",
+    "npm-run": "^5.0.1",
```

Comment:
There is a mix of hat versions and pinned versions, can we just use pinned versions and throw in renovate with the same configuration as other projects?

Reply:
fixed